### PR TITLE
Re-factor application-local.properties

### PR DESCRIPTION
**Pull request by phleven**
Re-factor application-local.properties

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -104,11 +104,11 @@
 			<artifactId>confusable-homoglyphs</artifactId>
 			<version>1.1.1</version>
 		</dependency>
-		<dependency>
+		<!-- <dependency>
```
Comment (cliffcaseyyet) [inline pom.xml:107]: It doesn't really make sense to me to comment out when you have git history. The previous change is permanently unloseable and any breaks can be figured out with blame. If we go this way we're going to end up with a codebase full of artifacts of every change all over.

File: src/main/resources/application-local.properties [hunk 2]
```diff
@@ -22,7 +22,7 @@ jasypt.encryptor.salt-generator-classname=org.jasypt.salt.RandomSaltGenerator
 jasypt.encryptor.iv-generator-classname=org.jasypt.iv.RandomIvGenerator
 jasypt.encryptor.string-output-type=base64
 
-server.use-forward-headers=true
+##server.use-forward-headers=true
```
Comment (cliffcaseyyet) [inline src/main/resources/application-local.properties:25]: i wonder if maybe we could just set it to false on the local props. might be cleaner

**Review by cliffcaseyyet — COMMENTED** [review]
As mentioned in services, please move the externalservices specific run docs into here.

File: README.md [hunk 3]
```diff
@@ -1,44 +1,32 @@
...
-# Requirements
-For building and running the elrrexternalservices you need:
-
-`Java >=1.8` : Download and install Java from here 
-* [Java](https://www.oracle.com/java/technologies/downloads/).
-
-`Maven >=3.6` : Download and install Apache Maven from here 
-* [Maven](https://maven.apache.org/) - Dependency Management.
-
-`Postgres >=11` : Download and install Postgres from here
```
Comment (cliffcaseyyet) [inline README.md:16]: this is still relevant. Remember, we are using the docker-compose but references to those dependencies should be explained.

Comment (cliffcaseyyet) [reply README.md:16]: I think for all of these you should explain that there are database and kafka dependencies, but then also say hey there's a repo with a docker-compose that resolves them locally. Remember there are two audiences - a developer and also someone deploying who needs to actually set those things up.

This should be the approach in all four repos.

**Review by cliffcaseyyet — APPROVED** [review]

**Comment by phleven** [comment]
Merged.

**Comment by phleven** [comment]
Merged,

**Pull request merged**